index.js

```diff
@@ -122,6 +124,17 @@ const Socket = exports.Socket = class TCPSocket extends Duplex {
     return this
   }
 
+  setKeepAlive (enable = false, initialDelay = 0) {
```

Comment:
This should support `setKeepAlive(initialDelay)`, i.e. if `typeof enable === 'number'` then `initialDelay = enable`.